### PR TITLE
Issue #15449: exlcuded rule462 from google-java-format.sh

### PR DESCRIPTION
**Pull request by Zopsss**
#15449 

Partially excluded rule462 from google-java-format.sh

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java [hunk 1]
```diff
@@ -3,9 +3,9 @@
 /** some javadoc. */
 public class InputWhitespaceAfterGood {
 
-  int xyz;       // multiple space between content and double slash.
+  int xyz; // multiple space between content and double slash.
   int abc; //       multiple space between double slash and comment's text.
-  int pqr;       //     testing both.
+  int pqr; //     testing both.
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java:8]: we lost this edge case after formatting. This edge case was explicitly mentioned in google style guide doc:

[4.6.2 Horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)

> 6. Between any content and a double slash (//) which begins a comment. Multiple spaces are allowed.
> 7. Between a double slash (//) which begins a comment and the comment's text. Multiple spaces are allowed.

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java:8]: should we revert this changes or not? @romani

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java:8]: Please revert this changes.
We need this violations.

We will use what formatter wants in InputFormatted.

**Comment by romani** [comment]
@Zopsss , please update PR

**Comment by Zopsss** [comment]
> @Zopsss , please update PR

done

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java]: if this the case then this section will be completely excluded

**Comment by Zopsss** [comment]
> @Zopsss , please update PR

done

**Review by romani — APPROVED** [review]
ok to merge.

just config changes, single review is fine.
